pelux.xml

```diff
@@ -12,80 +12,80 @@
 
   <!-- Base stuff -->
   <project remote="yocto"
-           upstream="rocko"
-           revision="9ed1178c87afce997d5a21cadae7461fb6bb48da"
+           upstream="sumo"
+           revision="bb91b2ae3ee5cf108aa2f9b78abb14d5aa00831d"
            name="poky"
            path="sources/poky"/>
 
   <project remote="oe"
-           upstream="rocko"
-           revision="352531015014d1957d6444d114f4451e241c4d23"
+           upstream="sumo"
+           revision="b0950aeff5b630256bb5e25ca15f4d59c115e7c1"
            name="meta-openembedded"
            path="sources/meta-openembedded"/>
 
   <!-- Pelagicore/Luxoft stuff -->
   <project remote="github"
-           upstream="master"
-           revision="eece6d9880676ad0e20fd200b248633b96efbee2"
+           upstream="sumo"
+           revision="4b25e519cde9a4d3f459fefb9ce692089324115d"
            name="Pelagicore/meta-bistro"
            path="sources/meta-bistro" />
 
   <project remote="github"
-           revision="7ee84f8f65c6d4733929ccffd35547d4ffd7c436"
-           upstream="master"
+           revision="88379880f8fe80379a6559686dd774dc6817b550"
+           upstream="sumo"
```

Comment:
Shouldn't the pelux-manifest on master point to the meta-pelux's latest revisions on master?